test/functional/test-position-observer.js

```diff
@@ -55,13 +54,12 @@ describes.realWin('PositionObserverApi', {
     posObserver = new PositionObserverApi(baseElement, iframe);
   });
 
-  it('should send one immediately wit correct value', () => {
-    const sendSpy = sandbox.spy(posObserver.subscriptionApi_, 'send');
-    posObserver.fire();
-    expect(sendSpy).to.be.calledOnce;
-    expect(sendSpy).to.be.calledWith('position', {
-      viewport: layoutRectLtwh(1, 2, 3, 4),
-      target: layoutRectLtwh(5, 6, 7, 8),
+  it('should get position with correct value', () => {
```

Comment:
Made this change , see [comments](https://github.com/ampproject/amphtml/pull/8708/files/ea827f64da0bb2a4bc6d19ea699ad3e1b63de474..40af82282974b5302a396a65a411a715a4d09bbc#r111459434). `fire()` will no longer covers the logic of `send one immediately without correct value`.